server/lib/article/assemble.js

```diff
@@ -18,14 +18,11 @@ const getCSS = require('./css');
 const environmentOptions = require('./environment-options');
 const handlebars = require('../handlebars');
 
-const getAuthors = data => {
-	const authors = (data.metadata || [])
-		.filter(item => !!(item.taxonomy && item.taxonomy === 'authors'))
-		.map(item => item.prefLabel);
-
+const getAuthors = article =>
 	// Somtimes there are no authors in the taxonomy. It's very sad but it's true.
-	return authors.length ? authors.join(', ') : (data.byline || '').replace(/^by\s+/i, '');
-};
+	article.authorConcepts.length
+		? article.authorConcepts.join(', ')
+		: (article.byline || '').replace(/^by\s+/i, '');
```

Comment:
From t'app, editorial are quite particular about the order of authors in the byline, and the concepts aren't sorted the same way - so probably best to byline first, concepts after...

Reply:
(we have a bug in the app backlog for this now, where we'll likely need to show byline and string replace in the author concept links 😱 )

Reply:
this isn't shown to users. i don't think it's shown anywhere, it's just required for a valid schema.org `NewsArticle`. the actual displayed byline does exactly that replace

Reply:
Fair enough!  Looks like there's an error if there's no authorConcepts but approachwise looks good then.